conda/environments/cugraph_dev_cuda10.1.yml

```diff
@@ -20,7 +20,7 @@ dependencies:
 - clang=8.0.1
 - clang-tools=8.0.1
 - cmake>=3.12
-- python>=3.6,<3.8
+- python>=3.6,<3.9
```

Comment:
So, are we supporting one more python version (adding 3.8 without dropping 3.6)?

Reply:
If yes, this looks good to me.

Reply:
we are adding 3.8.  I have left 3.6 in there for now, but it will be dropped soon